Podcast/RecastSubjectView.swift

```diff
@@ -0,0 +1,128 @@
+        backgroundColor = .offWhite
+
+        episodeMiniView = EpisodeMiniSubjectView()
+        episodeMiniView.delegate = self
+        addSubview(episodeMiniView)
+
+        recastBlurb = UILabel()
+        recastBlurb.textColor = .charcoalGrey
+        recastBlurb.font = ._14RegularFont()
+        recastBlurb.numberOfLines = 3
+        recastBlurb.isUserInteractionEnabled = true
+        recastBlurb.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(readPress)))
+        addSubview(recastBlurb)
+
+        separator = UIView()
+        separator.backgroundColor = .paleGrey
+        addSubview(separator)
+
+        recastBlurb.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(padding)
+            make.trailing.equalToSuperview().inset(padding)
+            make.top.equalToSuperview().inset(padding)
+        }
+
+        episodeMiniView.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(padding)
+            make.trailing.equalToSuperview().inset(padding)
+            make.top.equalTo(recastBlurb.snp.bottom).offset(padding)
+        }
+
+        separator.snp.makeConstraints { make in
+            make.top.equalTo(episodeMiniView.snp.bottom).offset(padding)
+            make.height.equalTo(separatorHeight)
+            make.leading.trailing.equalToSuperview()
+            make.bottom.equalToSuperview()
+        }
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setup(with episode: Episode, for user: User, isExpanded: Bool = false) {
+        recastBlurb.numberOfLines = 3
+        episodeMiniView.setup(with: episode)
+        let recastText = UserEpisodeData.shared.getBlurb(for: EpisodeToUser(episodeID: episode.id, userID: user.id)) ?? ""
+        recastBlurb.text = recastText
+        expandedText = NSMutableAttributedString(string: recastText, attributes: [NSAttributedStringKey.font: UIFont._14RegularFont()])
+        expandedText.append(NSMutableAttributedString(string: " Read Less", attributes: [NSAttributedStringKey.font: UIFont._14RegularFont(), NSAttributedStringKey.foregroundColor: UIColor.sea]))
+        expand(isExpanded)
+        if recastText == "" {
+            episodeMiniView.snp.remakeConstraints { make in
+                make.leading.trailing.equalToSuperview().inset(padding)
+                make.top.equalTo(recastBlurb.snp.bottom)
+            }
+        } else {
+            episodeMiniView.snp.remakeConstraints { make in
+                make.leading.trailing.equalToSuperview().inset(padding)
+                make.top.equalTo(recastBlurb.snp.bottom).offset(padding)
+            }
+        }
+    }
+
+    @objc func readPress() {
+        currentlyExpanded = !currentlyExpanded
+        expand(currentlyExpanded)
+        delegate?.expand(currentlyExpanded)
+    }
+
+    func expand(_ isExpanded: Bool) {
+        currentlyExpanded = isExpanded
```

Comment:
why do you set `currentlyExpanded = !currentlyExpanded`, then call expand on currentlyExpanded and then reset the variable here? i think that's repeating setting the variable 2x

Reply:
lol this was probably 4 am code